AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -204,62 +204,60 @@ def MaterialEditor_BasicFunctionalityChecks_AllChecksPass():
 
         # 10. Select the lighting background and verify the selection succeeded.
         neutral_urban_asset_path = os.path.join(
-            "@gemroot:MaterialEditor@", "Assets", "MaterialEditor", "LightingPresets",
-            "neutral_urban.lightingpreset.azasset")
+            azlmbr.paths.engroot, "Gems", "Atom", "Tools", "MaterialEditor", "Assets", "MaterialEditor",
+            "LightingPresets", "neutral_urban.lightingpreset.azasset")
```

Comment:
A helper function or const for the LightingPresets directory would be helpful, especially if this is used in other scripts. If this is a one-off, it's probably overkill.